CMakeLists.txt

```
@@ -90,7 +90,7 @@ endif(MYSQLPP_FOUND)
 #----------------------------------------------------------------------------
 # Boost
 #
-find_package(Boost COMPONENTS program_options filesystem system REQUIRED)
+find_package(Boost COMPONENTS program_options system REQUIRED)
```

Per https://github.com/JeffersonLab/QwAnalysis/commit/0fd95c24bf6530f0f9e5a9a6dcc5392e16bbcc03 we only included `system` as a dependency of `filesystem`, so this can be removed too now.
```suggestion
find_package(Boost COMPONENTS program_options REQUIRED)
```